src/scripts/release.zig

```diff
@@ -615,6 +618,54 @@ fn publish_node(shell: *Shell, info: VersionInfo) !void {
     });
 }
 
+fn publish_docker(shell: *Shell, info: VersionInfo) !void {
+    var section = try shell.open_section("publish docker");
+    defer section.close();
+
+    try shell.project_root.setAsCwd();
+    assert(try shell.dir_exists("dist/tigerbeetle"));
+
+    try shell.exec(
+        \\docker login --username tigerbeetle --password {password} ghcr.io
+    , .{
+        .password = try shell.env_get("GITHUB_TOKEN"),
+    });
```

Comment:
@cb22 just to highlight this login part, as I have never run this thing even once and don't know what it should look like.

Reply:
It seems good to me! Technically any process on the system could read `password` via `ps` or so, but on a dedicated runner where we would have bigger problems if compromised, I think that's OK.

The other way is to use `--password-stdin` and pipe it in.